### ISSUE TITLE
Synchronize water waves

### ISSUE THREAD

**Issue by lopezloo** [multitheftauto/mtasa-blue#2536]
**Is your feature request related to a problem? Please describe.**
Water waves aren't synced between players.

**Describe the solution you'd like**
Synchronize water waves.

**Additional context**
Possibly could be done by synchronizing `CWaterLevel::m_nWaterTimeOffset` (0xC228A4) variable + patching water functions to be FPS independent.

**Comment by Pirulax** [comment]
Yeah that should work, although even then the water will get out of sync after some time (so periodical syncs are required).
I think.
